### PR TITLE
Add disabled prop to ActionMenu items

### PR DESCRIPTION
**Pull request by vlaux**
#### What is the purpose of this pull request?
Simple as described on title. Details on linked issue #1150 

#### How should this be manually tested?
https://styleguide-git-fix-1150disabled-menu-options.styleguide-core.vercel.app/#/Components/Forms/ActionMenu

#### Screenshots or example usage
image
image

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [X] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Menu/index.js [hunk 1]
```diff
@@ -174,6 +174,7 @@ class Menu extends Component {
                     className={menuHeight ? 'overflow-auto' : ''}>
                     {options.map((option, index) => (
                       <button
+                        disabled={option.disabled}
                         key={index}
                         className="flex justify-between items-center t-body ph6 h-regular pointer hover-bg-muted-5 ma0 bg-transparent bn w-100 tl"
```
Comment (estacioneto) [inline react/components/Menu/index.js:179]: The button still has the `pointer` class when disabled.
Is it necessary for a button at all? 🤔  My guess is that you could remove it just fine.

Comment (estacioneto) [reply react/components/Menu/index.js:179]: Maybe hover classes should be removed on disabled too. What do you think?

Comment (vlaux) [reply react/components/Menu/index.js:179]: Agreed on removing `pointer`! But not sure about hover 🤔

File: react/components/Menu/index.js [hunk 2]
```diff
@@ -174,8 +174,11 @@ class Menu extends Component {
                     className={menuHeight ? 'overflow-auto' : ''}>
                     {options.map((option, index) => (
                       <button
+                        disabled={option.disabled}
                         key={index}
-                        className="flex justify-between items-center t-body ph6 h-regular pointer hover-bg-muted-5 ma0 bg-transparent bn w-100 tl"
+                        className={`flex justify-between items-center t-body ph6 h-regular ma0 bg-transparent bn w-100 tl ${
+                          option.disabled ? '' : 'hover-bg-muted-5 pointer'
+                        }`}
```
Comment (estacioneto) [inline react/components/Menu/index.js:181]: ⛏️  `classnames`? 😁

Comment (vlaux) [reply react/components/Menu/index.js]: Ohhh yes! :drake-yes:

**Pull request merged**